laravel-app/app/Http/Controllers/PostController.php

```diff
@@ -62,10 +63,11 @@ public function store(UserPostRequest $request): RedirectResponse
     /*
     * Display specific post.
     */
-    public function show(UserPost $post): View
+    public function show(UserPost $post, PostComment $comment, User $user): View
```

Comment:
modified it sir and tested , its working at this state 💯 I'm not sure if creating an instance of PostComment is the proper way tho
![image](https://github.com/bpocrafael/microblog-project/assets/144191092/312bd6a9-8c58-4323-8dc2-770aa4bff534)

Reply:
why?? still do that? i already suggested a code.

Reply:
sorry sir, I'm still getting used to using the format $post->comments().  I modified it and tested it, it works at this configurations.
![image](https://github.com/bpocrafael/microblog-project/assets/144191092/c8f5fb5a-a8a2-4c3d-a066-8282f71dae56)

Reply:
nope. still wont do

Reply:
![image](https://github.com/bpocrafael/microblog-project/assets/144989239/6b5b8608-652e-4f45-b8c4-e80aba3a87bf)

this is enough

Reply:
hmmmm I'll get back on it sir hehe for the meantime, i'll try resolving other concerns

Reply:
remember the post is already have an associate with the comment and also user?

why do you need to get those 2 and add it to the view.

we can just use the user and the comments in the post itself

Reply:
ohhh i thought sir kasi is by explicitly adding the other arguments eh mas madaling matrace or matrack if ano po yung napapass sa view. tho now I realized since may association na rin, it makes sense not to include

Reply:
what you doing is just adding a another query and another unnecessary codes.